style.css

```diff
@@ -0,0 +1,376 @@
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+body {
+  font-family: 'Inter', sans-serif;
+  display: flex;
+  align-content: center;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  background-color: #e5e5e5;
+}
+
+.background {
+  height: max-content;
+  width: auto;
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.container {
+  width: 327px;
+  height: 474px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  flex-direction: column;
+  margin-bottom: 2rem;
```

Comment:
- [ ] I would kindly suggest your your website responsiveness t make it responsive on all mobile devices The website should be responsive, following these breakpoints:

Minimum mobile screen size should be 375px ensure to 
Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels

https://github.com/Nessrine88/-Protofolio-/assets/92458236/d6d61c9d-cd14-455f-a357-4a69ebe8bf45